airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java

```diff
@@ -28,13 +28,13 @@ public class SnowflakeSqlGenerator implements SqlGenerator<SnowflakeTableDefinit
   private final ColumnId CDC_DELETED_AT_COLUMN = buildColumnId("_ab_cdc_deleted_at");
 
   @Override
-  public StreamId buildStreamId(String namespace, String name, String rawNamespaceOverride) {
+  public StreamId buildStreamId(final String namespace, final String name, final String rawNamespaceOverride) {
     // TODO
     return new StreamId(namespace, name, rawNamespaceOverride, StreamId.concatenateRawTableName(namespace, name), namespace, name);
   }
 
   @Override
-  public ColumnId buildColumnId(String name) {
+  public ColumnId buildColumnId(final String name) {
     // TODO
```

Comment:
what's TODO?

Reply:
nothing 🤦 I'm sure we'll get some beta users reporting weird character handling, but in my current testing this seems to work fine. I'll remove the comments.